docs/command/mongocli-cloud-manager-performanceAdvisor-slowOperationThreshold-disable.txt

```diff
@@ -1,66 +0,0 @@
-.. _mongocli-cloud-manager-performanceAdvisor-slowOperationThreshold-disable:
```

Comment:
is this command atlas only? why is it in cloud-manager?

Reply:
This is the response GuSS gave yesterday.
> I think everything inside internal/cli/mongocli is atlas commands that we "froze" for some reason

I also don't know why they're in cloud-manager. They're atlas only: https://github.com/mongodb/mongodb-atlas-cli/blob/20ee56764799694003370b88cf81a4312236cc6e/internal/store/atlas/performance_advisor.go#L99